janitor/functions.py

```diff
@@ -1811,6 +1888,81 @@ def remove_columns(
     return df.drop(columns=column_names)
 
 
+def get_occurrences(df: iter, og_index=None) -> pd.DataFrame:
+    """
+    This is a helper function for our remove_dupes function.
+    This function will return the first occurrences based on
+    the order of a passed in iterator.
+
+    :param df: This is our iterable object
+
+    :param og_index: This is our index to be added back to the DataFrame
+
+    :return: We return a pandas DataFrame object
+
+    """
+    test_list = {}
+    for row in df:
+        if not (row in test_list):
+            test_list[row] = 0
+    df = pd.DataFrame.from_dict(test_list.keys())
+    if og_index is not None:
+        df = df.set_index(og_index)
+    return df
+
+
+def remove_dupes(df: pd.DataFrame, keep="first") -> pd.DataFrame:
```

Comment:
I don't mean here to undo work already done, but I think we might not need this function, as the exact functionality desired is available in `pandas` native itself. (I checked on [the docs](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.drop_duplicates.html).) Definitely this is my fault for not keeping the issue tracker refreshed; #209 should have been closed a long time ago.

Reply:
**Action item:** Please remove the function `remove_dupes` and the helper function `get_occurrences`. The functionality is already covered in `pandas`.